roles/ceph-defaults/defaults/main.yml

```diff
@@ -352,6 +352,10 @@ cephfs_pools:
 
 ## OSD options
 #
+is_hci: false
```

Comment:
@neha-ojha you have to run `./generate_group_vars_sample.sh` to update sample files in `group_vars/`

Reply:
added it.